features/addressing/managing_zones/adding_zone.feature

```diff
@@ -0,0 +1,43 @@
+@addressing
+Feature: Adding a new zone with country type members
+    In order to apply taxes and allow shipping to geographical areas
+    As an Administrator
+    I want to add a new zone
+
+    Background:
+        Given I am logged in as an administrator
+        And the store has country "France"
+        And the store has a province "Alabama" with code "AL"
```

Comment:
I think that it would be much better to placed this in separated configuration, because we don't have to specify all of this zones to fulfill requirements of each feature